packages/api-specification/src/window/window-api.md

```diff
@@ -56,13 +56,45 @@ A reference to the new window.
 * `addListener(event, callback)` - Add an event listener to execute a given callback
 * `blur()` - Remove focus from the window. Returns a promise that resolves to nothing
 * `close()` - Close the window. Returns a promise that resolves to nothing
+* `flashFrame(flag)` - Flashes the window’s frame and taskbar icon.
```

Comment:
Think we need to put that they all/most return a promise and what they resolve to. If they all resolve to nothing, it might be worth saying something like this above the list: `All methods return a promise that resolves to nothing, unless stated otherwise`